docs/Makefile

```diff
@@ -87,9 +87,9 @@ qthelp:
 	@echo
 	@echo "Build finished; now you can run "qcollectiongenerator" with the" \
 	      ".qhcp project file in $(BUILDDIR)/qthelp, like this:"
-	@echo "# qcollectiongenerator $(BUILDDIR)/qthelp/cookiecutterproject_name.qhcp"
+	@echo "# qcollectiongenerator $(BUILDDIR)/qthelp/Invenio-Search-UI.qhcp"
```

Comment:
Do we change these to the project name?

Reply:
If yes I didn't spot it in my review of https://github.com/inveniosoftware/invenio-records-ui/pull/75

Reply:
Yes, you can usually ``git grep cookiecutter`` in the project root and there should be no hits.

Reply:
I will make a PR to fix inveniosoftware/invenio-records-ui#75